packages/lesswrong/server/editor/make_editable_callbacks.ts

```diff
@@ -270,6 +300,28 @@ function addEditableCallbacks<T extends DbObject>({collection, options = {}}: {
     return newDocument
   })
 
+  getCollectionHooks(collectionName).createAfter.add(
```

Comment:
I would honestly check for if collectionName === "Posts" before adding this hook. It seems aesthetically right, and will prevent stupid things from happening if someone adds a debate field to some other collection.